/public/home/hysplit/software/anaconda3/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  return f(*args, **kwds)
/public/home/hysplit/software/anaconda3/lib/python3.6/site-packages/cartopy/vector_transform.py:140: UserWarning: Some vectors at source domain corners may not have been transformed correctly
  u, v = target_proj.transform_vectors(src_crs, x, y, u, v)
1035.3511
548.20355
[500 580 660 740 820 900]
[950]
[ 950  970  990 1010 1030 1050 1070 1090 1110 1130]
1036.9834
550.57776
[500 580 660 740 820 900]
[950]
[ 950  970  990 1010 1030 1050 1070 1090 1110 1130]
